### PR TITLE
8272131: PhaseMacroExpand::generate_slow_arraycopy crash when clone null CallProjections.fallthrough_ioproj

### PR DESCRIPTION
**Pull request by huishi-hs**
Please help review this backport for 8272131: C2 crash in PhaseMacroExpand::generate_slow_arraycopy because arraycopy node's _ioproj_fallthrough is NULL.

Conflicts when backport
In jdk18: field name is fallthrough_ioproj , and it is in PhaseMacroExpand's CallProjections class.
In jdk11: field name is _ioproj_fallthrough, and it is directly in PhaseMacroExpand class.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8272131](https://bugs.openjdk.java.net/browse/JDK-8272131): PhaseMacroExpand::generate_slow_arraycopy crash when clone null CallProjections.fallthrough_ioproj ⚠️ Issue is not open.

### Reviewers
 * [Paul Hohensee](https://openjdk.java.net/census#phh) (@phohensee - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk11u-dev pull/250/head:pull/250` \
`$ git checkout pull/250`

Update a local copy of the PR: \
`$ git checkout pull/250` \
`$ git pull https://git.openjdk.java.net/jdk11u-dev pull/250/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 250`

View PR using the GUI difftool: \
`$ git pr show -t 250`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk11u-dev/pull/250.diff">https://git.openjdk.java.net/jdk11u-dev/pull/250.diff</a>

</details>

### CODE REVIEW

**Comment by huishi-hs** [comment]
I followed instructions in https://wiki.openjdk.java.net/display/SKARA/Backports. But wonder why backport lable is added automatically.

```
$ git checkout -b backport-5a526c1c
$ git fetch https://github.com/openjdk/jdk 5a526c1c5716f6d9a7fc94741bcdb2f424d342df
$ git cherry-pick --no-commit FETCH_HEAD
$ # resolve conflicts
$ git commit -m 'Backport 5a526c1c5716f6d9a7fc94741bcdb2f424d342df'
$ git push -u origin backport-5a526c1c
```

**Review by phohensee — APPROVED** [review]

**Comment by huishi-hs** [comment]
@phohensee Thanks for your review.

**Comment by RealCLanger** [comment]
@huishi-hs: This PR was integrated before jdk11u approval was given in JBS. No problem in this case - I have retroactively approved it. But please take care to wait for JBS approval next time.  (@phohensee: As sponsor, you should probably also check that approval was given. Sponsoring to me includes guiding non-comitters who contribute only occasionally or for the first time through the process 😉)  Thanks.

**Comment by phohensee** [comment]
Hmm. I had thought it was tagged jdk11u-fix-yes before I pushed. My apologies.

**Comment by huishi-hs** [comment]
@RealCLanger I'll check JBS  jdk11u-fix-yes before integration.

**Pull request closed without merging**
